models/DataObject/Service.php

```diff
@@ -1513,6 +1513,12 @@ public static function enrichLayoutDefinition(&$layout, $object = null, $context
         if (method_exists($layout, 'getChildren')) {
             $children = $layout->getChildren();
             if (is_array($children)) {
+                // Send information when we have block or simmilar element
```

Comment:
Branch base should be not `10.6`, it's already EOL.

Just noticed a typo there
```suggestion
                // Send information when we have block or similar element
```